ui/src/shared/components/ResizeContainer.js

```diff
@@ -1,11 +1,22 @@
 import React, {PropTypes} from 'react'
 import ResizeHandle from 'shared/components/ResizeHandle'
+import classNames from 'classnames'
```

Comment:
prefer `classnames` capitalization to differentiate from React's `className` special prop